subprojects/language-groovy/src/main/java/org/gradle/api/tasks/javadoc/Groovydoc.java

```diff
@@ -95,6 +95,7 @@ public Groovydoc() {
     @TaskAction
     protected void generate() {
         checkGroovyClasspathNonEmpty(getGroovyClasspath().getFiles());
+        getProject().delete(getDestinationDir());
```

Comment:
We could use `FileUtils.cleanDirectory()` here instead. Effectively it's the same, but it seems more correct and a tad faster.

Reply:
So the task would fail if the destination directory is a file? I guess it needs to exist since we create it as part of our task execution infrastructure, right?

Reply:
Done.